src/redecomp/transfer/MatrixTransfer.cpp

```diff
@@ -304,7 +304,6 @@ MPIArray<int> MatrixTransfer::buildRecvTrialElemDofs( const RedecompMesh& trial_
 {
   auto recv_trial_elem_dofs = MPIArray<int>( &getMPIUtility() );
 
-  auto rank = getMPIUtility().MyRank();
```

Comment:
was this a bug? Looks like "n_ranks" was already being used below.

Reply:
It's a simplification